cms/templates/cms/home_page.html

```diff
@@ -145,7 +145,7 @@ <h4>Pursue a Master’s</h4>
         </div>
       </section>
       <section class="row current-programs-section">
-        <h3>MITx MicroMasters Programs</h3>
+        <h3>MITx MicroMasters&reg; Programs</h3>
```

Comment:
The mark is only required once on the page, and the first appearance is above this one. Please remove this.